lib/mixlib/shellout/windows/core_ext.rb

```diff
@@ -308,25 +309,66 @@ def create(args)
           end
 
           unless bool
-            raise SystemCallError.new("CreateProcessAsUserW (You must hold the 'Replace a process level token' permission)", FFI.errno)
+            raise SystemCallError.new("CreateProcessAsUserW (You must hold the 'Replace a process level token' permission. Restart your machine after adding this right to make it effective.)", FFI.errno)
```

Comment:
The machine does not necessarily need to be restarted. Rather the user's logon context must be refreshed. This could be accomplished by logging out of windows and signing back in.